libs/nestjs-admin/src/adminSite.ts

```diff
@@ -97,6 +97,26 @@ class DefaultAdminSite {
         if (isDecimalType(column.type)) {
           cleanedValues[property] = parseFloat(value)
         }
+        if (isBooleanType(column.type)) {
+          if (column.isNullable) {
+            switch (value) {
+              case 'true':
+                cleanedValues[property] = true
+                break
+              case 'false':
+                cleanedValues[property] = false
+                break
+              case 'null':
+                cleanedValues[property] = null
+                break
+            }
+          } else {
+            // False boolean values aren't sent from the frontend by default
+            // To work around this, we send the false value then override it with a true value, in an array
```

Comment:
"Due to checkboxes not sending any value on submit, we need a to send a default value. If the checkbox is unchecked we'll only have the default, if it is checked we'll have an array with the default and the 'checked' value"